recipe/meta.yaml

```diff
@@ -13,6 +13,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
+  noarch: python
   number: 1
```

Comment:
Please bump the build number as well.

Reply:
My last commit fixes this issue, sorry about that